#!/usr/bin/env python
PACKAGE = "obstacle_avoiding_robot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("obstacle_threshold", int_t, 0, "The Distance At Which An Object Is Treated As An Obstale in cm", 20,  0, 50)
gen.add("turn_delay", double_t, 0, "Turning Delay In Seconds", 0.5, 0, 5.0)

exit(gen.generate(PACKAGE, "robot_control", "Parameters"))
